docs/examples/zerotrust/huproxy.md

```diff
@@ -0,0 +1,151 @@
+# HUProxy
+
+## What is HUProxy
+
+[HTTP(S)-Upgrade Proxy](https://github.com/google/huproxy) — Tunnel anything
+(but primarily SSH) over HTTP websockets.
```

Comment:
This file in general is great... however I would not call it 'zerotrust', but vpnless. As you are actually fully trusting the proxy. I know that it is becoming common to mix those two, but for me zero trust also implies end to end ecryption.

Reply:
Renamed per your request.